src/noise-models.lisp

```diff
@@ -0,0 +1,271 @@
+;;;; noise-models.lisp
+;;;;
+;;;; Author: Sophia Ponte
+
+(in-package #:qvm)
+
+;;; Implementation of rule-based noise models for the QVM.
+
+;;; These values are relatively aribtrary limits for NOISE-PREDICATE
+;;; priorty. They are used to order noise models for matching to
+;;; instructions.
+(defconstant +MAX-PRIORITY+ 20)
+(defconstant +MIN-PRIORITY+ 0)
+(alexandria:define-constant +DEFAULT-NOISE-PREDICATE-NAME+ "NOISE-NAME" :test #'string=)
```

Comment:
No need to capitalize:
```suggestion
(alexandria:define-constant +default-noise-predicate-name+ "NOISE-NAME" :test #'string=)
```

Reply:
should +MAX-PRIORTY+ and +MIN-PRIORITY+ also be lower case?

Reply:
I'd vote for that!